labs/nifty.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: nifty
+"parent: Labs"
+---
+# Lab Name
+nifty
+# Short Description
+nifty will be a low-code, metaprogrammable, standards sensitive and highly secure implementation to issue and trade NFTs, in a protocol agnostic way. 
+nifty code will address market place utilities for issuance and trading of NFTs. 
+
+# Scope of Lab
+nifty will be a low-code, metaprogrammable, standards sensitive and highly secure implementation to issue and trade NFTs, in a protocol agnostic way. 
+nifty code will address market place utilities for issuance and trading of NFTs. 
+nifty implementation will allow for extensibility and openness. An initial reference Hyperledger Fabric implementation is planned. 
+
+The initial code will be written in go with use of go-generate for metaprogramming
+
+For NFTs, the definition of fungibility is negated; that is, each token is unique and one cannot be exchanged for another. 
+
+A software system for the creation and trading for NFTs has to account for a way to tie the NFT to the unique object at issuance (Identity, authorization and a global registry); a marketplace for trading NFTs, which means establishing a price for the NFT as well as to transfer ownership. This means an implementation of these actions through interfaces supported by NFT token attributes and metadata.
+
+The following is the initial plan. 
+  Adopt a token standard (a combination of the TTF and ERC- inspired standards)  
+  Implement a marketplace (recode fabric samples )
+  Use Hyperledger Fabric as the underlying utility. See how ERC standards have been implemented in Fabric.
+  A complex systems view (Identity, distribution, decentralization, fairness, payment systems, cross-border, multi-protocol, interoperability of tokens, standards)- See under extensions.
```

Comment:
Recommend adding `*` in front of these lines so that they read as bullets.

Reply:
Can this infrastructure protect against the scam?